### ISSUE TITLE
HTML is not valid

### ISSUE THREAD

**Issue by henare** [AOGPN/opengovernment.org.au#109]
https://validator.w3.org/nu/?doc=https%3A%2F%2Fopengovernment.org.au%2F

**Comment by henare** [comment]
CSS is the same http://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fopengovernment.org.au%2F&profile=css3&usermedium=all&warning=1&vextwarning=&lang=en

**Comment by henare** [comment]
WCAG 2.0 should be pretty easy to fix http://achecker.ca/checker/index.php
